cfme/networks/floating_ips.py

```diff
@@ -0,0 +1,85 @@
+import attr
+from navmazing import NavigateToAttribute
+
+from cfme.common import WidgetasticTaggable
+from cfme.exceptions import ItemNotFound
+from cfme.modeling.base import BaseCollection, BaseEntity, parent_of_type
+from cfme.networks.views import FloatingIpDetailsView, FloatingIpView
+from cfme.utils import version
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+
+
+@attr.s
+class FloatingIp(WidgetasticTaggable, BaseEntity):
+    """Class representing floating ips"""
+    in_version = ('5.8', version.LATEST)
+    category = "networks"
+    page_name = 'floating_ip'
```

Comment:
**REQUIRED**: This can go, I think it is a left over from the old navigation system. I should probably strip it all out.